### DISCUSSION TITLE
Is there a way to set different permissions per world?

### DISCUSSION THREAD

**Discussion by Kyllingene** [Pryaxis/TShock#2147] [Q&A]
I want to have two worlds for my homebrew server; Creative and Survival. However, if I just name them that, people could come over, use /item, /godmode, etc. (allowed for Creative), and cheat the game, which is NOT what I want. So, short of duplicating the server files and running each separately, or making several permission groups and having everyone register twice, is there a way to set permissions for individual worlds?

**Comment by Kojirremer** [comment]
The RegionTrigger plugin had a feature which gave temporary permissions while being within a region, and since regions are tied to a world's ID, that's probably a good alternative.
No idea if there is any 1.4 build for RegionTrigger, though.

**Reply by Kyllingene** [reply]
Ok. I'll have to look into that. Thanks for the tip!
